python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -459,23 +491,37 @@ def __dropFunction( self, event ) :
 
 	def __dragEnter( self, widget, event ) :
 
-		return self.__dropFunction( event ) is not None
+		if self.__dropFunction( event ) is not None :
+			self.__dragEnterPointer = GafferUI.Pointer.getCurrent()
```

Comment:
It looks like this is the only place __dragEnterPointer is set.  I was trying to figure out if there is any way that it could be left at an out of date value when we need it, but I guess we are guaranteed that __dragEnter will be run before we need it, even in weird cases like a drag to self.

Reply:
Correct - you can't get into `__dragLeave`, `__dragMove` or `__drop` without first going through `__dragEnter`.